packages/next/build/webpack/loaders/next-babel-loader.js

```diff
@@ -16,7 +16,9 @@ module.exports = babelLoader.custom(babel => {
       }
       const loader = Object.assign({
         cacheCompression: false,
-        cacheDirectory: true
+        cacheDirectory: true,
+        // increment 'a' to invalidate cache
+        cacheIdentifier: 'next-cache-' + 'a'
```

Comment:
This should still take into account user configuration, and `next.config.js` changes I think.

Reply:
Mostly user config though, next.config.js isn't likely to change Babel

Reply:
I updated it to account for external babel changes but left off changes in `next.config.js` since I think it would be better to not invalidate every change to `next.config.js`